OpenRA.Game/Game.cs

```diff
@@ -383,9 +383,9 @@ static string ChooseShellmap()
 
 		// Note: These delayed actions should only be used by widgets or disposing objects
 		// - things that depend on a particular world should be queuing them on the worldactor.
-		static ActionQueue delayedActions = new ActionQueue();
-		public static void RunAfterTick(Action a) { delayedActions.Add(a); }
-		public static void RunAfterDelay(int delay, Action a) { delayedActions.Add(a, delay); }
+		static volatile ActionQueue delayedActions = new ActionQueue();
+		public static void RunAfterTick(Action a) { delayedActions.Add(a, Game.RunTime); }
+		public static void RunAfterDelay(int delay, Action a) { delayedActions.Add(a, Game.RunTime + delay); }
```

Comment:
Minor nit/request: Could you please rename `delay` &rarr; `delayMilliseconds` or similar.  I had to check to the code because I initially assumed that delay would be in ticks.